src/common/feedGenerator.ts

```diff
@@ -222,6 +231,20 @@ const wordsToFilters = ({
   );
 };
 
+const usersToFilters = ({
```

Comment:
Do we need this, we can just use result from sql?

Reply:
I wanted to do this one, in case we introduce the opposite side of the query at one stage?
Also don't think the one level fn loop will add much right?

Reply:
Yeah, agreed